### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [florisboard/florisboard#1311] [General]
Would a Mastodon account for this project be possible?

**Comment by patrickgold** [comment]
Possible? Yes. Is there really a need for it? Not really. The problem is that this adds an additional channel for communication to the already stressed system (I only have limited free time and not infinite sadly) and would be difficult for me to cope with this as well.

As an FOSS outlet/communication channel FlorisBoard already has a matrix group, which for now is sometimes also used for announcements besides discussion.
